app/src/renderer/components/governance/PageGovernance.vue

```diff
@@ -67,8 +67,15 @@ export default {
     showModalPropose: false
   }),
   computed: {
-    // TODO: get min deposit denom from gov params
-    ...mapGetters([`proposals`, `filters`, `bondingDenom`, `connected`])
+    ...mapGetters([
+      `proposals`,
+      `filters`,
+      `connected`,
+      `governanceParameters`
+    ]),
+    depositDenom() {
+      return this.governanceParameters.parameters.deposit.min_deposit[0].denom
```

Comment:
min_deposit can be several denoms?

Reply:
LOL yeah, idk why tho

Reply:
Can we put that into a getter as it is used several times?

Reply:
hmm ideally we'd like to support multiple deposit denoms for proposals in the future, so we'd need to have something like an array of them

Reply:
maybe leave it for another issue when we support multiple coins everywhere?

Reply:
I mean put `depositDenom` in a getter as you use it in too places. Can return an array when we have multiple denom min fees.

Reply:
updated